tox.ini

```diff
@@ -5,13 +5,13 @@ basepython = python3.9
 
 [baseenv]
 deps =
-    -rrequirements.txt
+    -rrequirements/requirements.txt
```

Comment:
I think space is missing in here, after the `-r` flag

Reply:
doesn't need, you can test it in your local machin